.github/workflows/atlas-pr.yml

```diff
@@ -8,22 +8,43 @@ on:
       - "atlas/**"
 
 jobs:
-  atlas_build:
-    name: Atlas Build
+  atlas_tests:
+    name: Atlas Tests
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest]
         node-version: [12.x]
       fail-fast: true
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - name: Use Node.js ${{matrix.node-version}}
         uses: actions/setup-node@v1
         with:
           node-version: ${{matrix.node-version}}
-      - name: build
+      - name: Install modules
         working-directory: ./atlas
-        run: |
-          yarn install --frozen-lockfile
-          yarn build
```

Comment:
We don't need to do build here anymore because it's built by Netlify anyway, right? If a build fails we should get a failed check on the PR?

Reply:
Correct! If there is an error during the build process, it will get cached by Netlify build process, and the preview will show as failed.

Reply:
> it will get cached by Netlify build process

You mean that we will be able to see the build error?